packages/rum-core/src/domain/requestCollection.ts

```diff
@@ -39,6 +39,7 @@ export interface RequestCompleteEvent {
   status: number
   responseType?: string
   startClocks: ClocksState
+  resolveDuration?: Duration
   duration: Duration
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should it be optional?
```suggestion
  duration?: Duration
```

Reply:
Good point, I will extend this type to make sure we reflect when `RequestCompleteEvent` contains and does not contain duration